src/components/Header/Header.tsx

```diff
@@ -0,0 +1,84 @@
+import { FC, useState, FormEvent } from 'react';
+import classNames from 'classnames';
+import { Error } from '../../types/Error';
+import { Todo } from '../../types/Todo';
+import { USER_ID, addTodo } from '../../api/todos';
+import { TodoCondition } from '../../types/TodoCondition';
+
+type Props = {
+  containsActive: boolean,
+  handleError: (err: Error) => void,
+  setTodoCondition: (arg: TodoCondition) => void,
+  onTrickTempTodo: (newTodo: Todo | null) => void,
+  setTodos: (prev: any) => void,
```

Comment:
why any here?
looks like should be Todo[]

Reply:
and for useState setter can be type like that
Dispatch<SetStateAction<Todo[]>>

Dispatch and SetStateAction are imports from 'react'